cli/cliui/agent.go

```diff
@@ -35,230 +25,217 @@ func Agent(ctx context.Context, writer io.Writer, opts AgentOptions) error {
 	if opts.FetchInterval == 0 {
 		opts.FetchInterval = 500 * time.Millisecond
 	}
-	if opts.WarnInterval == 0 {
-		opts.WarnInterval = 30 * time.Second
+	if opts.Wait && opts.FetchLogs == nil {
+		return xerrors.Errorf("fetch logs required when waiting for agent")
```

Comment:
nit: I'm not sure if I understand what this error means or whether it is actionable. Users may have trouble understanding it too.